tools/clang/test/CodeGenSPIRV/fn.param.inout.storage-class.hlsl

```diff
@@ -9,12 +9,17 @@ void foo(in float a, inout float b, out float c) {
 
 void main(float input : INPUT) {
 // CHECK: %param_var_a = OpVariable %_ptr_Function_float Function
+// CHECK: %param_var_b = OpVariable %_ptr_Function_float Function
+// CHECK: %param_var_c = OpVariable %_ptr_Function_float Function
 
 // CHECK: [[val:%\d+]] = OpLoad %float %input
 // CHECK:                OpStore %param_var_a [[val]]
-// CHECK:  [[p0:%\d+]] = OpAccessChain %_ptr_Uniform_float %Data %int_0 %uint_0
-// CHECK:  [[p1:%\d+]] = OpAccessChain %_ptr_Uniform_float %Data %int_0 %uint_1
```

Comment:
Note: The spec does not allow passing the result of OpAccessChain to OpFunctionCall.
The code written in `CHECK` here is the result of the codegen **_before_** spirv-opt legalization/optimization. The spirv-opt legalization recognizes this pattern and properly inlines the function as needed.